mofacts/client/views/experiment/instructions.js

```diff
@@ -71,20 +70,10 @@ function currLockOutMinutes() {
     const lockoutMinutes = userLockout.lockoutMinutes;
     const lockoutTime = lockoutTimeStamp + lockoutMinutes*60*1000;
     const currTime = Date.now();
-    if(currTime < lockoutTime){
-      // lockout is still in effect, if the user is an admin, we will allow them to continue
-      const newLockoutMinutes = Math.ceil((lockoutTime - currTime)/(60*1000));
-      logLockout(newLockoutMinutes);
-      return newLockoutMinutes;
-    }
+    const newLockoutMinutes = Math.ceil((lockoutTime - currTime)/(60*1000));
+    return newLockoutMinutes;
   } else {
```

Comment:
where is the if for this?

Reply:
@JRustyHaner check this and explain

Reply:
oh yeah, duh

Reply:
This is in a function that is called to get the time in minutes of a lockout that remains. 

It previously included a reset to the timer every time it's called and is called multiple times in the page.

Since it is sometimes called before the user lockout is returned from the server (which until that value is returned, the lockout is false), the lockout would be reset because the frontend didn't know a previous one existed.

This function is called every 250 milliseconds when the timeout interval is active, so if the return value from the server   hadn't been received, it would return false and reset the timer.

Removing the intervals ability to reset the lockout timer fixes the issue, to where the lockout timer can only be set the first time this page is loaded.

Reply:
Im confused on where this race condition would come from. The user cannot be in instructions if Meteor.user() is not defined correct? Since that information is stored on the client in MiniMongo there should not be a race condition accessing the lockout information.